styles/components/featured-resources.css

```diff
@@ -50,36 +50,36 @@
   align-items: flex-start;
   display: grid;
   grid-gap: 0 30px;
-  grid-template-areas: 'image category'
```

Comment:
One of the bigger adjustments, changed it so that there are only two sections instead of many. I understand we lose potential configurability here – so really curious on your take if this doesn't make sense.

Reply:
This may actually break the layout for this component, as there's no grid area named `copy`, and the other items are losing their placement. What does this do again?